rtf_vis_tool/src/Components/MVOSummary.js

```diff
@@ -0,0 +1,86 @@
+/* Component to render Multiview Optimizer Metrics. Specifically rotation averaging and
+translation averaging metrics.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Local Imports.
+import '../stylesheets/MVOSummary.css'
+
+function MVOSummary(props) {
+    /*
+    Args:
+        props.json.rotation_averaging_angle_deg.median_error (float): Median error of rotation averaging.
+        props.json.rotation_averaging_angle_deg.min_error (float): Min error of rotation averaging.
+        props.json.rotation_averaging_angle_deg.max_error (float): Max error of rotation averaging.
```

Comment:
I prefer: `Max angular error on global rotation for any frame after rotation averaging (pre-BA)`